client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/item-row.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import TextField from 'woocommerce/woocommerce-services/components/text-field';
+import {
+	setCustomsItemDescription,
+	setCustomsItemTariffNumber,
+	setCustomsItemOriginCountry,
+} from 'woocommerce/woocommerce-services/state/shipping-label/actions';
+import {
+	getShippingLabel,
+	isLoaded,
+	getFormErrors,
+} from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
+import CountryDropdown from 'woocommerce/woocommerce-services/components/country-dropdown';
+
+const TariffCodeTitle = localize( ( { translate } ) =>
+	<span>{ translate( 'Tariff Code' ) } (<a href="https://hts.usitc.gov/"
+		target="_blank" rel="noopener noreferrer">{ translate( 'look up' ) }</a>)</span>
+);
+
+const ItemRow = ( props ) => {
+	const {
+		errors,
+		packageId,
+		productId,
+		translate,
+		description,
+		defaultDescription,
+		tariffNumber,
+		originCountry,
+		countriesData,
+	} = props;
+
+	return <div className="customs-step__item-row">
+		<TextField
+			id={ packageId + '_' + productId + '_description' }
+			className="customs-step__item-description-column"
+			title={ translate( 'Description' ) }
+			value={ description }
+			placeholder={ defaultDescription }
+			updateValue={ props.setCustomsItemDescription }
+			error={ errors.description } />
+		<CountryDropdown
+			id={ packageId + '_' + productId + '_originCountry' }
+			className="customs-step__item-country-column"
+			title={ translate( 'Origin country' ) }
+			value={ originCountry }
+			updateValue={ props.setCustomsItemOriginCountry }
+			countriesData={ countriesData } />
+		<TextField
+			id={ packageId + '_' + productId + '_tariffNumber' }
+			className="customs-step__item-code-column"
+			title={ <TariffCodeTitle /> }
+			value={ tariffNumber }
+			updateValue={ props.setCustomsItemTariffNumber }
+			error={ errors.tariffNumber } />
+	</div>;
+};
+
+ItemRow.propTypes = {
+	siteId: PropTypes.number.isRequired,
+	orderId: PropTypes.number.isRequired,
+	packageId: PropTypes.string.isRequired,
+	productId: PropTypes.number.isRequired,
+	description: PropTypes.string.isRequired,
+	defaultDescription: PropTypes.string.isRequired,
+	tariffNumber: PropTypes.string.isRequired,
+	originCountry: PropTypes.string.isRequired,
+	errors: PropTypes.object,
+	countriesData: PropTypes.object.isRequired,
+	setCustomsItemDescription: PropTypes.func.isRequired,
+	setCustomsItemTariffNumber: PropTypes.func.isRequired,
+	setCustomsItemOriginCountry: PropTypes.func.isRequired,
+};
+
+const mapStateToProps = ( state, { orderId, siteId, productId } ) => {
+	const loaded = isLoaded( state, orderId, siteId );
```

Comment:
Can we call `loaded` something like `isShippingLabelLoaded`? Alternatively, we could import `isLoaded` using an alias so we can use `isLoaded` here as the var name.

Reply:
Changed in https://github.com/Automattic/wp-calypso/pull/24974/commits/9ed81d444905b55fd7bb01274144c1c89d1b456e. I went with renaming the variable, I'm not a fan of aliasing imports unless necessary.